### DISCUSSION TITLE
HRRR in Zarr Format

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/Herbie#8] [General]
A limited amount of HRRR data is available in Zarr format on [AWS](https://registry.opendata.aws/noaa-hrrr-pds/) in the bucket `s3://hrrrzarr/`. For some examples on accessing the data, check out the following

1. [🌟 Taylor Gowan's HRRR-Zarr notebooks](https://github.com/taylorgowan/zarr) <- Great place for getting started.
2. [Kevin Tyle's: HRRR-Zarr for AMS Short Course](https://github.com/ktyle/python_pangeo_ams2021/blob/main/notebooks/03_Pangeo_HRRR.ipynb)
1. My [notebook](https://github.com/blaylockbk/HRRR_archive_download/blob/master/notebooks/zarr_HRRR.ipynb) has an example of my initial testing.
1. Some twitter chatter on how to access the HRRR Zarr data with xarray:
https://twitter.com/DJGagneDos/status/1365345126306955267?s=20
1. [Randy Chase showing New York Temperature Analysis](https://colab.research.google.com/drive/1SGu28q8DEDi4_281F0_O0Lr9cYCPcJjT?usp=sharing)

There is also this closed issue with some potential useful details to glean: https://github.com/blaylockbk/HRRR_archive_download/issues/2

**Comment by scollis** [comment]
So are there plans to integrate this with Herbie
I was talking to @mgrover1 this morning and I think we should start a conversation on the Pangeo Discourse

**Reply by blaylockbk** [reply]
I would love to integrate this with Herbie. I'm not actively working on this, but @amotl is and has some ideas.
See #153 and #155 

One thing I'd like to see is a list of other datasets that are available in Zarr format and their source.

**Reply by scollis** [reply]
Great. I will check out those

**Comment by scollis** [comment]
It would be good to be able to have the data "lazy loaded" so only once you do an operation it downloads. So you can subset first
